### PR TITLE
Collectors signup

### PR DESCRIPTION
**Pull request by Valioesi**
I transferred the 3 pages of the sign up flow for collectors. Also fixed the image upload (new chosen image was not shown and arrow not rendered) and external links (not opening due to href not being set). 

To test: goto /sammeln/anmelden and walk through the flow.

### CODE REVIEW

File: components/Forms/SignUp/index.tsx [hunk 1]
```diff
@@ -374,18 +490,55 @@ const validate = (
     errors.email = 'Wir benötigen eine valide E-Mail Adresse';
   }
 
-  if (!values.nudgeBox && !values.newsletterConsent) {
+  if (values.phoneNumber && !validatePhoneNumber(values.phoneNumber)) {
```
Comment (nmauersberg) [inline components/Forms/SignUp/index.tsx:493]: This was somehow not triggered, when entering 110

**Review by nmauersberg — CHANGES_REQUESTED** [review]
Testing it I entered 110 as my phone number and it failed only after submitting, but without an error message. I landed on the default error screen: "Ops, bei uns ist etwas schief gegangen".

The rest went fine. The new disabled fields are a nice improvement!

File: pages/sammeln/anmelden/index.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            <p>
+              Du willst an der Sammlung oder dem Event am{' '}
+              {formatDate(new Date(date))} in/am {location} teilnehmen?
+            </p>
+          ) : (
+            <p>Du willst uns bei der Sammlung unterstützen?</p>
+          )}
+
+          <SignUp
+            fieldsToRender={[
```
[the remark this answers is not in the document]
Comment (Valioesi) [reply pages/sammeln/anmelden/index.tsx:46]: Nice, thanks! Yeah, I added that already in the old project. Pretty useful, cause this way I could also easily implement the SmallSignup with only the email field

File: components/Forms/SignUp/index.tsx (hunk 1 above)
Comment (Valioesi) [reply components/Forms/SignUp/index.tsx:493]: Oh yeah, apparently I added a stricter phone validation (at least 6 digits) on the backend side. Maybe I forgot to add it in the frontend back then. So this bug was there in the old project as well. But probably not a huge deal so far.

**Comment by nmauersberg** [comment]
@Valioesi Cool, now it works for me.

But on the profile page the Image is too large. Is it the same for you?

Profile-Image

**Review by nmauersberg — APPROVED** [review]
The rest was fine for me. :)

**Pull request merged**